ds_judgements_public_ui/templates/includes/browse_by_court.html

```diff
@@ -5,10 +5,9 @@ <h2 id="browse-by-court" class="judgment-browse-by-court__header">Find judgments
 
       <ul class="judgment-browse-by-court__list">
         {% for court in courts %}
-        {# {% for court in courts | sort(attribute='name') %} #}
           <li class="judgment-browse-by-court__court">
             <span class="judgment-browse-by-court__court-name">
-                <a class="judgment-browse-by-court__court-link" href="/results?courts={{ court.code }}">
+                <a class="judgment-browse-by-court__court-link" href="/judgments/advanced_search?court={{ court.code }}">
```

Comment:
If you use the `url` template tag, this link won't break if (for any reason) our url scheme changes on the backend.

`href="{% url 'advanced_search' %}?court={{ court.code }}"`